sdks/java/core/src/main/java/org/apache/beam/sdk/annotations/Experimental.java

```diff
@@ -39,6 +39,7 @@
   ElementType.TYPE
 })
 @Documented
+@SuppressWarnings("checkstyle:JavadocMethod")
```

Comment:
Seems like it should have documentation. I suggest `You can indicate a category for the experimental feature. This is unused and serves only as a hint to the reader.`

Reply:
Ok. let me remove this one. I was misled by the @Documented annotation.